base-notebook/test/test_container_options.py

```diff
@@ -224,3 +238,28 @@ def test_container_not_delete_bind_mount(container, tmp_path):
     assert rv == 0 or rv["StatusCode"] == 0
     assert p.read_text() == "some-content"
     assert len(list(tmp_path.iterdir())) == 1
+
+
+# def test_jupyter_env_vars_to_unset_as_root(container):
+#     """Environment variables names listed in JUPYTER_ENV_VARS_TO_UNSET
+#     should be unset in the final environment."""
+#     c = container.run(
+#         tty=True,
+#         user="root",
+#         environment=[
+#             "JUPYTER_ENV_VARS_TO_UNSET=SECRET_ANIMAL,UNUSED_ENV,SECRET_FRUIT",
+#             "FRUIT=bananas",
+#             "SECRET_FRUIT=mango",
+#             "SECRET_ANIMAL=cats",
+#         ],
+#         command=[
+#             "start.sh",
+#             "bash",
+#             "-c",
+#             "echo I like $FRUIT and ${SECRET_FRUIT:-stuff}, and love ${SECRET_LOVE:-to keep secrets}!",
+#         ],
+#     )
+#     rv = c.wait(timeout=10)
+#     assert rv == 0 or rv["StatusCode"] == 0
+#     logs = c.logs(stdout=True).decode("utf-8")
+#     assert "I like bananas and stuff, and love to keep secrets!" in logs
```

Comment:
I need to keep this as a comment since I included the commit with the tests, but not yet the commit where `JUPYTER_ENV_VARS_TO_UNSET` was implemented.

Reply:
There is a better way to skip the test:

Just add decorator like this (with a good description):
```
@pytest.mark.skip(reason="no way of currently testing this")
```

Reply:
Great tip, thanks so much @mathbunnyru!